setup.py

```diff
@@ -30,6 +30,7 @@ def read(*parts):
         "Framework :: Django :: 2.2",
         "Framework :: Django :: 3.0",
```

Comment:
Django 3.0 is now EOL. Don't know if that's out of scope for this PR.

Reply:
I will manage it in separate PR